numba_dppy/dpnp_glue/dpnp_array_creations_impl.py

```diff
@@ -31,15 +31,21 @@ def common_impl(a, b, out, dpnp_func, PRINT_DEBUG):
     sycl_queue = dpctl_functions.get_current_queue()
 
     b_usm = dpctl_functions.malloc_shared(b.size * b.itemsize, sycl_queue)
-    dpctl_functions.queue_memcpy(sycl_queue, b_usm, b.ctypes, b.size * b.itemsize)
+    event = dpctl_functions.queue_memcpy(
```

Comment:
Wouldn't it be better to create a function:

```python
def synchronous_memcpy(q, dst, src, size):
     event = dpctl_functions.queue_memcpy(q, dst, src, size)
     dpctl_functions.event_wait(event)
     dpctl_functions.event_delete(event)
```

and use it throughout?

Reply:
Opinions?

Reply:
About dpctl API I would like to propose to try to avoid braking changes:
```
def queue_memcpy(..., wait=True): 
    ...
    return event

def queue_memcpy_async(...):
    return queue_memcpy(..., wait=False)
```